src/test/java/core/basesyntax/service/impl/FruitShopServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.handler.BalanceOperationHandler;
+import core.basesyntax.strategy.handler.PurchaseOperationHandler;
+import core.basesyntax.strategy.handler.ReturnOperationHandler;
+import core.basesyntax.strategy.handler.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class FruitShopServiceImplTest {
+    private static Map<FruitTransaction.Operation, OperationHandler> operationHalndlerMap =
```

Comment:
Please check carefully the list of common mistakes before implementing.

<img width="910" alt="image" src="https://github.com/mate-academy/jv-fruit-shop-tests/assets/72043323/d390474b-3065-45b9-af73-ac29274908ec">

Fix needed ones and update PR. Thanks!

Reply:
Hope, I've correctly understood this comment and made needed changes in new PR